vendor/nomad/Makefile

```diff
@@ -0,0 +1,14 @@
+## Package - nomad
```

Comment:
```suggestion
## Package - nomad
export PACKAGE_ENABLED = false
```

@zloeber please add this line at the top of `Makefile` to skip packaging due to apk build fail issue. See build logs here: https://g.codefresh.io/build/5e4ec06cc5b02577f39bc133?step=apk_build_3_11&tab=output
and related root of the problem here: https://github.com/hashicorp/nomad/issues/5535